internal/testutil/acc/cluster.go

```diff
@@ -27,14 +28,11 @@ type ClusterInfo struct {
 // GetClusterInfo is used to obtain a project and cluster configuration resource.
 // When `MONGODB_ATLAS_CLUSTER_NAME` and `MONGODB_ATLAS_PROJECT_ID` are defined, creation of resources is avoided. This is useful for local execution but not intended for CI executions.
 // Clusters will be created in project ProjectIDExecution.
-func GetClusterInfo(tb testing.TB, req *ClusterRequest) ClusterInfo {
+func GetClusterInfo(tb testing.TB, req *ClusterRequest, specs ...admin.ReplicationSpec) ClusterInfo {
 	tb.Helper()
 	if req == nil {
 		req = new(ClusterRequest)
 	}
```

Comment:
do you think GetClusterInfo tests are useful or it's ok just to test the formattters?

Reply:
No needed IMO.
Only differences are to use an existing project/cluster or not